plugins/extractit/icon_extractit/actions/url_extractor/action.py

```diff
@@ -16,4 +16,8 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        return {Output.URLS: clear_urls(extract(Regex.URL, params.get(Input.STR), params.get(Input.FILE)))}
+        return {
+            Output.URLS: clear_urls(
+                extract(Regex.URL, params.get(Input.STR), params.get(Input.FILE), params.get(Input.KEEP_ORIGINAL_URLS))
```

Comment:
Add default value to params.get, 
```suggestion
                extract(Regex.URL, params.get(Input.STR), params.get(Input.FILE), params.get(Input.KEEP_ORIGINAL_URLS, False))
```